pkg/reconciler/resource/knservice_test.go

```diff
@@ -43,8 +45,8 @@ func TestNewServiceWithDefaultContainer(t *testing.T) {
 		EnvVar("TEST_ENV2", "val2"),
 		Label("test.label/2", "val2"),
 		ServiceAccount("god-mode"),
-		Requests(resource.MustParse("250m"), resource.MustParse("100Mi")),
-		Limits(resource.MustParse("250m"), resource.MustParse("100Mi")),
+		Requests(&cpuRes, &memRes),
+		Limits(&cpuRes, nil),
```

Comment:
In `deployment_test.go` this is set to

```golang
Requests(&cpuRes, &memRes),
Limits(&cpuRes, &memRes),
```

I don't care which one we go for, be I think we should keep both test cases identical, for consistency and to ensure they both yield the same result given the same input.